source/docs/3/advanced_raster_analysis.rst

```diff
@@ -0,0 +1,176 @@
+Advanced Raster Analysis (QGIS3)
+=================================
+
+For city planning, the most useful raster data is Land-Cover. This data informs a wide variety of strategic planning activities. To do this, QGIS has advanced raster capabilities built-in via the ``Raster Calculator``. In this tutorial, we will explore the options available for styling categorical raster and the functionality provided by the raster calculator.
```

Comment:
This tutorial is not about raster calculator. It is about advanced raster tools. Mention all the tools that are used.